### DISCUSSION TITLE
what happend ? just same code as in your solutions parking management document.

### DISCUSSION THREAD

**Discussion by cobaltautomationdev** [ultralytics/ultralytics#15715] [Q&A]
same code as https://docs.ultralytics.com/guides/parking-management/#python-code-for-parking-management
in windows10,  labels and texts all concentrated in the top of pic.
![image](https://github.com/user-attachments/assets/2264ea02-6dda-45d4-bfc9-37f13332f9b1)

but in google colab it runs very well. 
![image](https://github.com/user-attachments/assets/c210cbc5-be2d-44c6-b961-056ecb3fb310)

**Comment by glenn-jocher** [comment]
@cobaltautomationdev it looks like the issue might be related to the environment differences between Windows 10 and Google Colab. Please ensure you are using the latest version of the Ultralytics package. If the issue persists, try updating your graphics drivers and verifying your Python and library versions match those used in Colab.

**Reply by cobaltautomationdev** [reply]
An older version ,8.1.0 of the software runs very well in the same environment on my computer. 
Since version8.2.0 However, now even a simple image prediction is not working correctly; the labels and text are messed up.

![image](https://github.com/user-attachments/assets/5f714737-acfe-4696-bec2-1ddee218fb59)
after downgrade to 8.1.0
![image](https://github.com/user-attachments/assets/b8722352-1d5c-4386-adfb-a58a9c70ca0c)

**Reply by cobaltautomationdev** [reply]
https://github.com/ultralytics/ultralytics/issues/15049 just noticed that your team already know this issue

**Reply by glenn-jocher** [reply]
Thank you for pointing that out. We are aware of the issue and are actively working on a fix. Please stay tuned for updates in the linked issue.
